app/controllers/api/v1/classifications_controller.rb

```diff
@@ -34,6 +34,14 @@ def project
     index
   end
 
+  def offset
+    @controlled_resources = Classification.where("project_id = ? AND id > ?", params[:project_id], params[:last_id])
+    .includes(:project)
+    .includes(:user)
+    render json_api: serializer.page(params, controlled_resources, context),
```

Comment:
I think you'll want to replicate what i did with incomplete and use the index action with the `scope_for` setup on the Classification model to fit into the default authorization rules to make sure non-project owners can't access other project classifications.